### DISCUSSION TITLE
New stat screen page for skills?

### DISCUSSION THREAD

**Discussion by JesterWizard** [MokhaLeee/fe8u-cskillsys-kernel#279] [General]
Considering we can go up to 23 skills now, I'd say it's worth considering a 5th page for them (RAM allowing) as we can only show up to 8 currently on the screen. Or is there not enough free space to do this?

Alternatively, maybe the names can be removed to allow more space for the icons without creating an extra page? That would allow enough room to fit all 23. 

![image](https://github.com/user-attachments/assets/e50ef1c5-e3c7-4031-bfbe-14dc962236f2)

**Comment by MokhaLeee** [comment]
I"m not quite approval of page expansions,but it may be a priper plan that contributors pull up their plan on page4 and their rtext page（and the code also needs review）and let the users deside which plan to use by themselves via designers' config patch. Maybe later I may pull a commit on this.

**Comment by MokhaLeee** [comment]
https://github.com/MokhaLeee/fe8u-cskillsys-kernel/pull/281
